aws-geo-location/src/main/java/com/amplifyframework/geo/location/AWSLocationGeoPlugin.kt

```diff
@@ -54,6 +61,9 @@ class AWSLocationGeoPlugin(
     private val defaultMapName: String by lazy {
         configuration.maps!!.default.mapName
     }
+    private val defaultSearchIndexName: String by lazy {
+        configuration.searchIndices!!.default
```

Comment:
nit: prevent using not-null operator.

Reply:
https://github.com/aws-amplify/amplify-android/pull/1475#discussion_r701255383
discussed in the past